apcd-cms/src/apps/exception/templates/exception_submission_form/exception_other_form.html

```diff
@@ -165,6 +165,13 @@ <h4>Acknowledgment of Terms</h4>
               $('#expiration-date').attr('max', maxDate);
           });
         </script>
+      <!-- To transform email to lowercase -->
+        <script>
+          const emailInput = document.querySelector('input[type="email"]');
+          emailInput.addEventListener("input", function(e) {
+            e.target.value = e.target.value.toLowerCase();
+          });
+        </script>
```

Comment:
I believe a simpler way to do this and avoid duplicate code would be to have something in the `_clean_email` function in the `apcd_database.py` file. Perhaps something like `email = email.toLowerCase() `at the start of the function? Let me know what you think

Reply:
We could also just add a re.IGNORECASE flag to the end of the already existing _clean_email util. Sorry, I think I misunderstood and thought you wanted this check to stay on the front end

Reply:
No worries! The main goal I had for this was to not save capital letters for email in the database. The re.IGNORECASE would work but it would still end up saving the capital letters if I am not mistaken. Where we need to perform this conversion can be up to preference. Open to thoughts from @fnets and @edmondsgarrett

Reply:
Updated to have the toLowerCase on the _clean_email util